src/App.jsx

```diff
@@ -0,0 +1,68 @@
+
+import './index.css';
+import {ProductPage} from "./pages/productsPage/productPage.jsx"
```

Comment:
You can import all components from one File!

Reply:
yeah did that now .